views/extras.tmpl

```diff
@@ -1,368 +1,358 @@
+			{{if eq $txType "merged_debit"}}
+				<td class="text-right">{{.MergedTxnCount}}</td>
+				<td class="text-right fs15">
+					{{template "decimalParts" (float64AsDecimalParts .SentTotal 8 false)}}
+				</td>
 
-              {{else if eq $txType "merged_credit"}}
-                <td class="text-right">{{.MergedTxnCount}}</td>
-                <td class="text-right fs15">
-                  {{template "decimalParts" (float64AsDecimalParts .ReceivedTotal 8 false)}}
-                </td>
+			{{else if eq $txType "merged_credit"}}
+				<td class="text-right">{{.MergedTxnCount}}</td>
+				<td class="text-right fs15">
+					{{template "decimalParts" (float64AsDecimalParts .ReceivedTotal 8 false)}}
+				</td>
 
-              {{else if eq $txType "merged"}}
-                <td class="text-right">{{.MergedTxnCount}}</td>
-                {{if .IsFunding}}
-                  <td class="text-right fs15">
-                    {{template "decimalParts" (float64AsDecimalParts .ReceivedTotal 8 false)}}
-                  </td>
-                  <td class="text-right">&mdash;</td>
-                {{else}}
-                  <td class="text-right">&mdash;</td>
-                  <td class="text-right fs15">
-                    {{template "decimalParts" (float64AsDecimalParts .SentTotal 8 false)}}
-                  </td>
-                {{end}}
+			{{else if eq $txType "merged"}}
+				<td class="text-right">{{.MergedTxnCount}}</td>
+				{{if .IsFunding}}
+				<td class="text-right fs15">
+					{{template "decimalParts" (float64AsDecimalParts .ReceivedTotal 8 false)}}
+				</td>
+				<td class="text-right">&mdash;</td>
+				{{else}}
+				<td class="text-right">&mdash;</td>
+				<td class="text-right fs15">
+					{{template "decimalParts" (float64AsDecimalParts .SentTotal 8 false)}}
+				</td>
+				{{end}}
 
-              {{else if or (eq $txType "credit") .IsFunding}}  <!-- .IsFunding = true && txType = "all" is a credit -->
-                <td class="text-right fs15">
-                  {{template "decimalParts" (float64AsDecimalParts .ReceivedTotal 8 false)}}
-                </td>
-                {{if ne .MatchedTx ""}}
-                    <td class="text-right"><a href="/tx/{{.MatchedTx}}/in/{{.MatchedTxIndex}}"
-                      data-txid="{{.MatchedTx}}"
-                      data-action="mouseover->address#hashOver mouseout->address#hashOut"
-                      >spent</a></td>
-                {{else}}
-                    <td class="text-right">unspent</td>
-                {{end}}
+			{{else if or (eq $txType "credit") .IsFunding}}  <!-- .IsFunding = true && txType = "all" is a credit -->
+				<td class="text-right fs15">
+					{{template "decimalParts" (float64AsDecimalParts .ReceivedTotal 8 false)}}
+				</td>
+				{{if ne .MatchedTx ""}}
+				<td class="text-right"><a href="/tx/{{.MatchedTx}}/in/{{.MatchedTxIndex}}"
+					data-txid="{{.MatchedTx}}"
+					data-action="mouseover->address#hashOver mouseout->address#hashOut"
```

Comment:
This may seem like madness, but there were a ton of spaces and the html is ~2 kB smaller just for the `addressTable` template.

Reply:
It's only madness because github treats a tab as 8 spaces, which is... excessive, IMO. Looks great in my IDE.